docs/reference/python.md

```diff
@@ -1,3 +1,5 @@
 # Python
 
-TBD
+SQLMesh's Python API reference is located [here](https://sqlmesh.readthedocs.io/en/stable/_readthedocs/html/sqlmesh.html).
+
+The SQLMesh [CLI interface](./cli.md) calls Python [`Context` class methods](https://sqlmesh.readthedocs.io/en/stable/_readthedocs/html/sqlmesh/core/context.html) on the backend. The CLI implementation code can be viewed [on Github](https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/cli/main.py).
```

Comment:
Nit: "CLI" already contains the word "Interface", so we can just say "The SLQMesh CLI [...]" I think.

Reply:
Good catch